src/components/shareable-url-modal/published-view/published-view.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import classnames from 'classnames';
+import UrlBox from '../url-box/url-box';
+import Button from '../../ui/button';
+import Dropdown from '../../ui/dropdown';
+import MenuOption from '../../ui/menu-option';
+
+import { getFilteredPlatforms, handleResponseUrl } from '../utils';
+
+const PublishedView = ({
+  hostingPlatformLocalStorageVal,
+  hostingPlatforms,
+  onChange,
+  onCopyClick,
+  onRepublishClick,
+  platform,
+  showCopied,
+}) => {
+  const platformsKeys = Object.keys(hostingPlatformLocalStorageVal);
+  const platformsVal = Object.values(hostingPlatformLocalStorageVal);
+
+  const url = platform
+    ? hostingPlatformLocalStorageVal[platform]['endpoint']
+    : platformsVal[0]['endpoint'];
```

Comment:
`published-view` only appears after atleast one successful publish ? If not, does platformsVal always contain atleast 1 element ?

Reply:
in the published-view, yes it always contains at least 1 element as there's a condition to only render the publishedView if the length of value we get from localStorage is greater than 1